src/test/groovy/life/qbic/datamodel/dtos/business/CostEstimateSpec.groovy

```diff
@@ -46,9 +46,6 @@ class CostEstimateSpec extends Specification {
 
     def "Missing optional Field definitions shall haven null values in a CostEstimate object"() {
 
-        given:
-        CostEstimateId costEstimateId = new CostEstimateId("ab", "cd", 1)
-
         when:
         Offer testOffer =
                 new Offer.Builder(customer, projectManager, "Archer", "Cartoon Series", [], selectedAffiliation).build()
```

Comment:
why do we have an offer here and not a cost estimate?

Reply:
Good Catch, i adapted the test and oversaw that line 👍